DOC-3039-41 Patch Release 4.7.2 (#503)
corda/corda-docs: content/en/docs/corda-enterprise/4.7/release-checksum-enterprise.md

> Before installing any downloaded file, we recommend that you verify the integrity of each file as a security precaution, to ensure that the files have not been tampered with or become corrupted during the download. You can do this by calculating the checksum of your downloaded file and then comparing its value to the checksum value provided below.
> 

Reviewer: ```suggestion
It is best practice to verify the integrity of each file you download. You can ensure the files have not been tampered with or corrupted during the download by calculating the checksum of your downloaded file, then comparing that value to the checksum value below.

```
This isn't part of your changes, but it's been bugging me for several patch releases now!

Reviewer 2: This isn't the right time to change this. I suggest we leave it as is for now, and maybe have a strategic discussion about the boilerplate text when we have more time.
